kv/kv.go

```diff
@@ -202,6 +203,9 @@ type LockCtx struct {
 	PessimisticLockWaited *int32
 	LockKeysDuration      *time.Duration
 	LockKeysCount         *int32
+	ReturnValues          bool
+	Values                map[string][]byte
+	ValuesLock            sync.Mutex
```

Comment:
I'm curious why needs Mutex here?

Reply:
The pessimistic lock RPC may be sent concurrently for multiple regions.